src/components/App/index.tsx

```diff
@@ -318,12 +318,14 @@ const App = () => {
     hub.current?.invoke('UpdateRoute', route.url);
   };
 
+  const skipEventKey = Routes.map((rt) => rt.skipEventKey && rt.url).includes(window.location.pathname);
+
   // Change what's rendered based on global application state
   let appContent: JSX.Element;
   if (!appContext || (!isAuthenticated && connection?.connectionStatus === undefined)
       || (isAuthenticated && appContext.event === undefined)) {
     appContent = (<div className={styles.infoText}>Loading...</div>);
-  } else if (isAuthenticated && appContext.event !== undefined) {
+  } else if ((isAuthenticated && appContext.event !== undefined) || skipEventKey) {
```

Comment:
most likely to break things, but I think it is good

Reply:
![image](https://github.com/FIRSTinMI/fim-queueing/assets/3743825/7ed2e39c-f72a-4603-8136-c4eb520649ef)